src/GitHub.InlineReviews/Services/PullRequestSession.cs

```diff
@@ -126,6 +117,34 @@ public string GetRelativePath(string path)
         }
 
         /// <inheritdoc/>
+        public async Task<IPullRequestReviewCommentModel> PostReviewComment(string body, string commitId, string path, int position)
```

Comment:
They're part of the PR - I had to move things around a bit in order to be able to get hold of an `IUsageTracker` at the right place.